app/src/main/java/com/mapbox/maps/testapp/examples/markersandcallouts/infowindow/InfoWindowActivity.kt

```diff
@@ -0,0 +1,105 @@
+package com.mapbox.maps.testapp.examples.markersandcallouts.infowindow
+
+import android.graphics.Bitmap
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.geojson.Point
+import com.mapbox.maps.MapView
+import com.mapbox.maps.dsl.cameraOptions
+import com.mapbox.maps.plugin.annotation.generated.*
+import com.mapbox.maps.plugin.gestures.OnMapLongClickListener
+import com.mapbox.maps.plugin.gestures.addOnMapLongClickListener
+import com.mapbox.maps.plugin.gestures.removeOnMapLongClickListener
+import com.mapbox.maps.testapp.R
+import com.mapbox.maps.testapp.utils.BitmapUtils
+import java.text.DecimalFormat
+
+/**
+ * Test activity showcasing using the legacy InfoWindow API above Washington D.C
```

Comment:
Since there is not more InfoWindow API IMHO there's nothing to showcase. I'd reworded saying that `Instead of old InfoWindow API you might use View Annotation + Point Annotation APIs`

Reply:
agree